external_parser/unit_tests/test_vw_external_parser.cc

```diff
@@ -359,4 +359,65 @@ BOOST_AUTO_TEST_CASE(cb_compare_dsjson_with_fb_models_deferred_actions_w_activat
   BOOST_CHECK_EQUAL_COLLECTIONS(buffer_fb_model.begin(), buffer_fb_model.end(),
                                 buffer_dsjson_model.begin(),
                                 buffer_dsjson_model.end());
+}
+
+BOOST_AUTO_TEST_CASE(cb_pdrop_05_parse) {
+  std::string input_files = get_test_files_location();
+
+  auto buffer =
+      read_file(input_files + "/valid_joined_logs/cb_joined_with_pdrop_05.fb");
```

Comment:
Can you document under `external_parser/unit_tests/test_files/README.md` how this file is generated?

Reply:
done